news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+I think it would be better to check for [YesMyDarknesss’s interview](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) because it was such a neat answer that there's not much to say for me. He's definitely a better fruit teacher than I am :)
+
+Shout out to my man <3
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Is there anything specific that keeps you motivated even after years of playing, and what do you think is the most important to not lose that motivation, even when you are not improving?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+In my opinion, focusing on enjoying *playing osu!catch* itself actually has been keeping me motivated.
+
+For example let's say I've reached to the point where I'm not improving and slowly get demotivated, then I still keep trying to play the maps which are easy or which have the song I like to listen to. 
+The more you push yourself - the more stressed you'll get, so don't be discouraged even if you are not improving. It's good mentally to relax yourself by playing easy stuff and enjoying it.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+As someone who was playing osu!catch for a long period of time, where do you see this game mode in the future? What do you feel may change?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+In osu!catch, the players in the top 50 hardly change over the years, because more and more people go inactive while there is a small inflow of new papers to form a new generation. This is saddening, but if this continues further, I don't think I see a good future for osu!catch T_T
```

Comment:
```suggestion
In osu!catch, the players in the top 50 hardly change over the years, because more and more people go inactive while there is a small inflow of new papers to form a new generation. This is saddening, but if this continues further, I don't think I see a good future for osu!catch T_T

```